var/spack/repos/builtin/packages/automake/package.py

```diff
@@ -39,8 +39,13 @@ def patch(self):
         if '@:1.15.1' in self.spec:
             files_to_be_patched_fmt = 't/wrap/{0}.in'
 
+        if '@1.16.3' in self.spec:
```

Comment:
Can you check if this:
```suggestion
        if self.spec.satisfies('@1.16.3:'):
```
works? It's likely that future versions would have this change too.

Reply:
It works well. But I think just changing `'@1.16.3'` to `'@1.16.3:'` would be enough.